server/controllers/finance/reports/financial.patient.handlebars

```diff
@@ -28,7 +40,10 @@
             <td class="text-right" title="{{this.trans_id}}">{{this.trans_id}}</td>
             <td style="max-width : 200px; white-space : nowrap; overflow : hidden; text-overflow : ellipsis;" title="{{this.description}}">{{this.description}}</td>
             <td class="text-right">
-              {{debcred this.balance ../metadata.enterprise.currency_id}}
+              {{debcred this.debit ../metadata.enterprise.currency_id}}
```

Comment:
According to Mr. Wayne, we should either have a single column with the `debcred` helper, or two columns with them separated.  Was this a feature request from IMCK?

HEV has been using this report for a number of months without the separate of debits and credits - having a single column allows more of the text to fit in the description, which is helpful to them.  If this wasn't an explicit feature request from IMCK, I would prefer we leave it as it was before.

Reply:
I'm going to do this, I'm going to restore the patient's financial status report, because the IMCK's request would be more about the financial situation of the employees